### PR TITLE
adding visdcc recipe

### PR DESCRIPTION
**Pull request by jfallmann**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/visdcc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% set name = "visdcc" %}
+{% set version = "0.0.40" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/6f/a3/289b8dec1459807ad6288b41447e9cdad44bff46ea97182a388710f52c7e/{{ name }}-{{ version }}.tar.gz
```
Comment (chrisburr) [inline recipes/visdcc/meta.yaml]: Can you use the predictable URL (see the example) so the bot can monitor for updates

Comment (jfallmann) [reply recipes/visdcc/meta.yaml]: @chrisburr  Sorry but this is the only URL I have, would also prefer a predictable one

Comment (chrisburr) [reply recipes/visdcc/meta.yaml]: [see the example](https://github.com/conda-forge/staged-recipes/blob/0d7438c23fb07a8075adfb70182f229f9870b8c5/recipes/example/meta.yaml#L14):
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment (jfallmann) [reply recipes/visdcc/meta.yaml]: Oh, had no idea, thx for the fix

**Pull request merged**